umake/__init__.py

```diff
@@ -85,6 +85,17 @@ def set_logging_from_args(args, parser):
         _setup_logging()
 
 
+def set_loading_frameworks(args):
```

Comment:
This would be better renamed as "should_load_all_frameworks()", returning a boolean, which is then chained outside to load_frameworks().

This is because:
* you don't really "set loading…" as you are loading here, so the name is wrong
* you don't use sys.argv() but arg itself, don't leak internal implementation in description as it's going to be certainly wrong in the future :)